src/components/landing/VideoHero/VideoHero.js

```diff
@@ -42,11 +42,11 @@ const VideoHero = () => {
               <div className="sm:text-center md:max-w-2xl md:mx-auto lg:col-span-7 lg:text-left">
                 <Headline className="tracking-right mt-12">
                   <span className="text-gray-900">Batteries included</span>
-                  <span className="block text-gray-900">Backstage</span>
+                  <span className="block text-gray-900">Internal Developer Portal</span>
```

Comment:
Can we go even better?

> [prefix] Internal Developer Portal

What's the best possible word or two that can go into `prefix`? 

Let's make a big change if we're going to make one.

Reply:
How about this from the vision statement:

Achieve ultimate developer effectiveness - Internal developer portal that unlocks your development potential and improves the maturity of software.

Reply:
Please review latest screenshots. Im not sure about the dot mesh thing.